http_cache/cache_invalidation.rst

```diff
@@ -84,6 +85,58 @@ to support the ``PURGE`` HTTP method::
         }
     }
 
+Then, register the class as a service that :doc:`decorates </service_container/service_decoration>`
+``http_cache``::
+
+.. configuration-block::
+
+    .. code-block:: yaml
```

Comment:
I would love to see a follow up PR against 6.2 adding how this can be done using an attribute 👍

Reply:
Done, see #17975.